docs/variants/asus_kgpe_d16/dasharo-compatibility/cbmem_boot_time_measurment.md

```diff
@@ -0,0 +1,61 @@
+# Booting time measurment with cbmem on Ubuntu OS
+
+## Coldboot time measurment
```

Comment:
Please refer this test suite to another test suite in `dasharo-compatibility`. All test descriptions have standarized design (permanent sections: Test description, Test configuration data, Test setup, Test steps and Expected results). These are mandatory corrections to be made before the task is completed.